### ISSUE TITLE
Parachains runtime release process 

### ISSUE THREAD

**Issue by muharem** [paritytech/cumulus#1657]
The document is Work In Process and can be altered. 

## Problem 
Parachains runtime release takes up to 2 weeks to prepare and publish. 

## Context 
Parachains runtime release is semi-automated and takes up to 2 weeks to prepare and publish. Most of this time are manual checks,  fixes, backports, weight calculation reruns. The target release cadence is a month which brings a manpower intensive release process.

In this document we will gather all related issues, and use the document as a reference for the problem.

## Goals
- parachains runtime release takes up to 3 days;
- issue-free release builds;

## Known issues
- manual storage migration checks - issue/TBD @muharem 
- manual xcm configs checks - issue/TBD @muharem 
- manual transaction_version check - issue/TBD @muharem 
- weight generation and review process - issue/TBD @muharem

**Comment by gilescope** [comment]
The transaction_version check is semi-automated and a newer version of that check is being worked on by @chevdor (this is the metadata differ that only needs the runtimes as inputs).

**Comment by chevdor** [comment]
Yes, I am actually working on an option that may even allow handling the `transaction_version` topic in a fully automated way.
